regolith/schemas.py

```diff
@@ -1169,12 +1169,12 @@
         "month": {
             "description": "the month when the entry was created",
             "required": False,
-            "type": "integer",
+            "anyof_type": ["integer", "string"],
         },
         "year": {
             "description": "the year when the entry was created",
             "required": False,
-            "type": "integer",
+            "anyof_type": ["integer", "string"],
```

Comment:
this should be integer only.  Months can be "May" or 5 but years can only be 2020

Reply:
Done